Wikipedia/Code/WMFAppViewController.m

```diff
@@ -530,17 +530,9 @@ - (void)showNearbyListAnimated:(BOOL)animated {
 
 #pragma mark - House Keeping
 
-- (void)performHousekeepingIfNecessary {
-    NSDate* lastHousekeepingDate        = [[NSUserDefaults standardUserDefaults] objectForKey:@"LastHousekeepingDate"];
-    NSInteger daysSinceLastHouseKeeping = [[NSDate date] daysAfterDate:lastHousekeepingDate];
-    //NSLog(@"daysSinceLastHouseKeeping = %ld", (long)daysSinceLastHouseKeeping);
-    if (daysSinceLastHouseKeeping > 1) {
-        //NSLog(@"Performing housekeeping...");
-        MWKDataHousekeeping* dataHouseKeeping = [[MWKDataHousekeeping alloc] init];
-        [dataHouseKeeping performHouseKeeping];
-        [[NSUserDefaults standardUserDefaults] setObject:[NSDate date] forKey:@"LastHousekeepingDate"];
-        [[NSUserDefaults standardUserDefaults] synchronize];
-    }
+- (void)performHousekeeping {
+    MWKDataHousekeeping* dataHouseKeeping = [[MWKDataHousekeeping alloc] init];
+    [dataHouseKeeping performHouseKeeping];
```

Comment:
so `MWKDataHouskeeping` isn't really a class. it's just a function. as a boyscout, can we at least extract `performHouseKeeping` into a method on `MWKDataStore`? this way it's just: `[self.dataStore performHouseKeeping]` and we can even test it!